modules/processing/targetinfo.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (C) 2010-2015 Cuckoo Foundation.
+# This file is part of Cuckoo Sandbox - http://www.cuckoosandbox.org
+# See the file 'docs/LICENSE' for copying permission.
+
+import os.path
+import logging
+import json
+
+from lib.cuckoo.common.abstracts import Processing
+from lib.cuckoo.common.config import Config
+from lib.cuckoo.common.integrations.file_extra_info import static_file_info
+from lib.cuckoo.common.integrations.parse_url import HAVE_WHOIS, URL
+from lib.cuckoo.common.objects import File
+from lib.cuckoo.common.constants import CUCKOO_ROOT
+
+log = logging.getLogger(__name__)
+processing_conf = Config("processing")
+
+class TargetInfo(Processing):
+    """General information about a file."""
+
+    #Added: Added function which uses pymisp to perform threat attribution
+    def compare_to_misp(self, apikey, url, sha256):
+        """Utilize threat intelligence platform MISP to perform threat attribution.
+        @return: threat actor (if applicable)
+        """
+        #Added: Added imports necessary for pymisp
+        try:
+            from pymisp import PyMISP
+            from pymisp import logger as pymisp_logger
+            pymisp_logger.setLevel(logging.ERROR)
+        except ImportError:
+            log.error("pip3 install pymisp")
```

Comment:
pymisp api is known to be broken on each update, so you need to specific version on which it works

Reply:
i put latest version of pymisp so if that is different let us know